_source/_posts/2022-04-21-jhipster-k8s-azure.md

```diff
@@ -0,0 +1,667 @@
+---
+layout: blog_post
+title: "JHipster Microservice Deployed to Azure with Cosmos DB"
+author: andrew-hughes
+by: contractor
+communities: [devops,security,java,javascript]
+description: ""
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image:
+type: awareness|conversion
+---
+#### JHipster Azure AKS Microservice
+
+In this tutorial, you're going to deploy a JHipster-based, reactive microservice to Azure Kubernetes Service (AKS). You'll use Azure's Cosmos DB as a persistent store for one of the services. For security, you'll use Okta as an OAuth and OIDC provider. You'll also securely encrypt all secrets in the project configuration files using Kubernetes secrets and `kubeseal`. This tutorial focuses on deploying an already generated project to Azure AKS. It does not go into great detail about generating the project. To see how the project was generated using JHipster, take a look at [this previous tutorial](https://developer.okta.com/blog/2021/01/20/reactive-java-microservices). 
+
+The project has a few different pieces:
+
+- JHipster registry: a Eureka server for service discovery and a Spring Cloud Config server for centralized configuration management
```

Comment:
```suggestion
- JHipster Registry: a Eureka server for service discovery and a Spring Cloud Config server for centralized configuration management
```
Since you capitalized it in the diagram.